package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "weatherJS",
+  "version": "1.0.0",
+  "description": "<p align=\"center\">\r   <img src=\"https://img.shields.io/badge/Microverse-blueviolet\">\r </p>",
+  "main": "index.js",
+  "dependencies": {
+    "file-loader": "^6.2.0",
+    "html-loader": "^2.1.2",
+    "html-webpack-plugin": "^5.3.1",
+    "webpack-dev-server": "^3.11.2"
+  },
+  "devDependencies": {
+    "webpack": "^5.35.0",
+    "webpack-cli": "^4.6.0"
+  },
+  "scripts": {
+    "build": "npx webpack",
+    "watch": "npx webpack --webpack"
```

Comment:
- [ ] The `watch` script it's not working right now
![Screenshot from 2021-04-21 18-36-25](https://user-images.githubusercontent.com/30977845/115624105-84369900-a2d0-11eb-9784-db89304f2bd3.png)

- [ ] The application stops working after running the build script.

**OPTIONAL**

- [ ] You can implement a script to make use of the `webpack-dev-server` dependency
```JS
"start": "webpack serve"
```

- Is there any particular reason why you are using npx in these scripts?